src/style.css

```diff
@@ -1 +1,115 @@
-/* add styles here */
+.html {
+  margin: 0;
+  padding: 0;
+}
+
+@font-face {
+  font-family: Avenir;
+  src:
+    url(./fonts/Avenir-Book.ttf) format('Avenir'),
+    url(./fonts/Avenir-Heavy.ttf) format('Avenir'),
+    url(./fonts/Avenir.ttc);
+}
+
+input {
+  border: none;
+  outline: 0;
+}
```

Comment:
Avoid styling HTML elements directly; use classes instead. This helps to avoid conflicts and makes the styles more reusable and modular.